scss/components/table.scss

```diff
@@ -59,6 +60,68 @@ $block: #{$fd-namespace}-table;
               @include fd-var-color("background-color", fd-color-state("selected-hover"), --fd-color-background-selected-hover);
             }
         }
+
+        &--valid,
+        &--warning,
+        &--error,
+        &--information {
+            border: none;
+            & .#{$block}__cell:first-child {
```

Comment:
interesting technique. does it work in IE?

Reply:
@bcullman sure does!

![Screen Shot 2019-09-17 at 10 09 21 AM](https://user-images.githubusercontent.com/29607818/65063550-5985a100-d933-11e9-9aee-3b9dc14b1967.png)